src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Square extends ColorSupplier implements Figure {
+    private static int side1;
+    private static int side2;

```

Comment:
The 'side1' and 'side2' variables should not be static as they represent the state of an individual square instance.